.github/workflows/test-external.yml

```diff
@@ -22,43 +24,59 @@ jobs:
         project:
           - repo: "morpho-org/morpho-data-structures"
             dir: "morpho-data-structures"
-            cmd: "halmos --function testProve --loop 4 --symbolic-storage"
+            cmd: "--function testProve --loop 4 --symbolic-storage"
```

Comment:
is the yices solver not used here due to the weird failure you mentioned?

Reply:
yes :(